packages/components/src/local/molecules/turn-progression/index.tsx

```diff
@@ -87,8 +87,9 @@ export const TurnProgression: React.FC<Props> = (props: Props) => {
   }
 
   const countup = (): any => {
+    const startTime = Math.round(new Date(adjudicationStartTime).getTime() / 1000)
     const now = Math.floor(new Date().getTime() / 1000)
-    const seconds = now - progressionState.startTime
```

Comment:
This was a strange issue, @lebaphi  & @lilitkarapetyan 

`progressionState` is the state of this form.  But, here inside this timer callback, it wasn't receiving the updated state. I _think_ that's because the timer callback gets defined when the timer is initiated, and that version of the function gets `cached`.  So, I had to generate the data I needed using form props, rather than form-state.

Things worked in the end.